src/pages/home/ReportScreen.tsx

```diff
@@ -709,6 +710,16 @@ function ReportScreen({route, currentReportID = '', navigation}: ReportScreenPro
         fetchReport();
     }, [fetchReport]);
 
+    useEffect(() => {
+        // If the linked action is previously available but now deleted,
+        // remove the reportActionID from the params to not link to the deleted action.
+        const isLinkedActionBecomesDeleted = prevIsLinkedActionDeleted !== undefined && !prevIsLinkedActionDeleted && isLinkedActionDeleted;
```

Comment:
NAB: Can we please call this `isLinkedActionNowDeleted` or `linkedActionWasDeleted`? My preference is for that second option. Since it's just a name change let's do this in a follow up please.

Reply:
Or even better, let's not use a variable at all and put this right in the if statement.